src/btree/bt_vrfy.c

```diff
@@ -434,7 +434,8 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             memset(&addr_unpack, 0, sizeof(addr_unpack));
             addr_unpack.newest_start_durable_ts = ckpt->start_durable_ts;
             addr_unpack.oldest_start_ts = ckpt->oldest_start_ts;
-            addr_unpack.oldest_start_txn = ckpt->oldest_start_txn;
+            addr_unpack.oldest_start_txn =
+              ckpt->write_gen > S2C(session)->base_write_gen ? ckpt->oldest_start_txn : 0;
```

Comment:
We usually use `WT_TXN_NONE` instead of an explicit 0 when assigning to transaction ids.

Reply:
Fixed, ty.